doc/classes/OS.xml

```diff
@@ -508,7 +508,8 @@
 			<argument index="0" name="tag_name" type="String">
 			</argument>
 			<description>
-				Returns [code]true[/code] if the feature for the given feature tag is supported in the currently running instance, depending on platform, build etc. Can be used to check whether you're currently running a debug build, on a certain platform or arch, etc. See feature tags documentation.
+				Returns [code]true[/code] if the feature for the given feature tag is supported in the currently running instance, depending on platform, build etc. Can be used to check whether you're currently running a debug build, on a certain platform or arch, etc. See the [url=https://docs.godotengine.org/en/3.1/getting_started/workflow/export/feature_tags.html]Feature Tags[/url] documentation.
```

Comment:
I would prefer a link to the `en/latest` docs instead of `en/3.1`, since this is the master branch. All links will then be changed to point to `en/3.2` before releasing 3.2-stable.